### PR TITLE
feat(pill): search pill variant

### PR DESCRIPTION
**Pull request by tr-ann**
Updated hover state of "filter" pill for [FX-3170](https://artsyproduct.atlassian.net/browse/FX-3170) ticket
[Figma](https://www.figma.com/file/0svJJjlcrCFBkVYzagKP3Z/FX-v3-Surfaces?node-id=4128%3A1534) spec
Jonathan [comment](https://artsyproduct.atlassian.net/browse/FX-3170?focusedCommentId=41304)

### Demo
image

### CODE REVIEW

**Comment by tr-ann** [comment]
@dzucconi I not fully understand how I can change only hover state because we just set css styles matches to variants [here](https://github.com/artsy/palette/blob/2f280e3f9a6dc287be22d75af98e2dc4e2336fb4/packages/palette/src/elements/Pill/Pill.tsx#L58). Can you help me with it?

File: packages/palette/src/elements/Pill/tokens.ts [hunk 1]
```diff
@@ -50,19 +50,23 @@ export const PILL_VARIANTS: Record<PillVariant, Record<PillState, any>> = {
       border-radius: 20px;
       height: 40px;
       padding: 0 ${themeGet("space.2")};
+      color: ${themeGet("colors.black60")};
```
Comment (dzucconi) [inline packages/palette/src/elements/Pill/tokens.ts:53]: This isn't correct according to the Figma spec

Comment (dzucconi) [reply packages/palette/src/elements/Pill/tokens.ts:53]: Spec = https://www.figma.com/file/m6gDpKHEWDbYJyrwsVZDBr/Artsy-3.0-Design-System?node-id=1020%3A781

Comment (tr-ann) [reply packages/palette/src/elements/Pill/tokens.ts:53]: But in [this](https://www.figma.com/file/0svJJjlcrCFBkVYzagKP3Z/FX-v3-Surfaces?node-id=4128%3A1220) spec default text color is `black60`

**Review by dzucconi — APPROVED** [review]

**Comment by tr-ann** [comment]
@dzucconi can I merge it?

**Comment by artsyit** [comment]
:rocket: PR was released in [`@artsy/palette@17.7.0`](https://github.com/artsy/palette/releases/tag/%40artsy/palette%4017.7.0) :rocket:

**Pull request merged**